Utilities/ocpasswordgen/ocpasswordgen.c

```diff
@@ -4,11 +4,48 @@
 **/
 
 #include <stdio.h>
+#ifdef _WIN32
+#include <conio.h>
```

Comment:
Need to test this on Windows before merging.

Reply:
I tested it before creating pull request, it works